lib/ext/heroku/command/addons.rb

```diff
@@ -550,42 +583,6 @@ def get_attachments(options = {})
       ).body
     end
 
-    def change_plan(direction)
-      addon_name, plan = args.shift, args.shift
-
-      if addon_name && !plan # If invocated as `addons:Xgrade service:plan`
-        deprecate("No add-on name specified (see `heroku help #{current_command}`)")
-
-        addon = nil
-        plan = addon_name
-        service = plan.split(':').first
-
-        action("Finding add-on from service #{service} on app #{app}") do
-          addon = resolve_addon(app, service)
-          addon_name = addon['name']
-        end
-        display "Found #{addon_name} (#{addon['plan']['name']}) on #{app}."
-      else
-        addon_name = addon_name.sub(/^@/, '')
-      end
-
-      raise CommandFailed.new("Missing add-on plan") if plan.nil?
-      raise CommandFailed.new("Missing add-on name") if addon_name.nil?
-
-      labels = {upgrade: 'Upgrading', downgrade: 'Downgrading'}
-      action("#{labels[direction]} #{addon_name} to #{plan}") do
```

Comment:
Between the `change_plan` method and the `upgrade` method, the two lines above are the only ones that changed. The words "upgrading" and "downgrading" simply don't appear any longer and are instead replaced with a generic "changing plan".